variable/technology/power-plant.yaml

```
@@ -0,0 +1,799 @@
+                plant category.
+   unit: Mcal
+
+Shut-Down Cost|Electricity|Oil:
+   description: Shutdown heat cost per power generation unit who belongs to the specified power
+                plant category.
+   unit: Mcal
+Start-Up Cost|Electricity|Coal:
+   description: Startup heat cost per power generation unit who belongs to the specified power
+                plant category.
+   unit: Mcal
+
+Start-Up Cost|Electricity|Gas:
+   description: Startup heat cost per power generation unit who belongs to the specified power
+                plant category.
+   unit: Mcal
+
+Start-Up Cost|Electricity|Nuclear:
+   description: Startup heat cost per power generation unit who belongs to the specified power
+                plant category.
+   unit: Mcal
+
+Start-Up Cost|Electricity|Oil:
+   description: Startup heat cost per power generation unit who belongs to the specified power
+                plant category.
+   unit: Mcal
+
+Variable Cost|Electricity|Biomass|Traditionnal:
+   description: variable cost of a typical Traditionnal Biomass power plant
+                biomass from local and pre-existing
+                resources mostly waste :domestic and agricultural waste, forest residues.
+   unit: €/MWh
+
+Variable Cost|Electricity|Biomass|New and imported :
+   description: variable cost of a typical new biomass power plant
+                biomass from dedicated energy crops and biomass imports.
+   unit: €/MWh
+
+Variable Cost|Electricity|Nuclear:
+   description: variable cost of a typical nuclear power plant
+   unit: €/MWh
+
+Variable Cost|Electricity|Gas|OCGT:
+   description: variable cost of a typical Open Cycle Gas Turbine power plant
+   unit: €/MWh
+
+Variable Cost|Electricity|Gas|Coal:
+   description: variable cost of a typical coal power plant
+   unit: €/MWh
+
+Variable Cost|Electricity|Lignite:
+   description: variable cost of a typical Lignite power plant
+   unit: €/MWh
+
+# Variables related to thermal power plants
+Number of Units|Electricity|Biomass|Traditional:
+   description: number of units of kind "Traditionnal Biomass power plant "
+                biomass from local and pre-existing
+                resources mostly waste :domestic and agricultural waste, forest residues.
+   unit:
+
+Number of Units|Electricity|Biomass|New and imported:
+   description: Number of units of kind "new biomass power plant "
+                biomass from dedicated energy crops and biomass imports.
+   unit:
+
+Number of Units|Electricity|Nuclear:
+   description: Number of units of kind "nuclear power plant "
+   unit:
+
+Number of Units|Electricity|Gas|OCGT:
+   description: Number of units of kind "Open Cycle Gas Turbine power plant"
+   unit:
+
+Number of Units|Electricity|Gas|Coal:
+   description: Number of units of kind "coal power plant "
+   unit:
+
+Number of Units|Electricity|Lignite:
+   description: Number of units of kind "Lignite power plant "
+   unit:
+Maximum Active power|Electricity|Biomass|Traditionnal:
```

please change throughout this document

```suggestion
Maximum Active Power|Electricity|Biomass|Traditionnal:
```